headers/validation.h

```diff
@@ -108,6 +109,52 @@ auto operator|(T1 /*unused*/, T2 /*unused*/) {
 
 enum Separator { Space, Newline };
 
+//this contains some specific code which would be available in c++20
+namespace C20 {
+
+constexpr int countl_zero(unsigned long long x) {
```

Comment:
do you have a source for this (please add as comment) or handwritten?

Reply:
this is handwritten